branchpro/apps/_dash_app.py

```diff
@@ -193,9 +193,11 @@ def parse_contents(self, contents, filename, is_si=False):
         if message is None:
             if not is_si:
                 if ('Time' not in data.columns) or (
-                        'Incidence Number' not in data.columns):
+                        ('Imported Cases' not in data.columns) and (
```

Comment:
Data files without an Incidence Number column can now be uploaded to the inference app, where they cause undesirable behavior. 

One solution may be to have this method accept as optional arguments two lists of column names, and require that all names in the first list are present and at least one name in the second list is present.